### DISCUSSION TITLE
Customize Background Image not working

### DISCUSSION THREAD

**Discussion by kartishp** [tailwindlabs/tailwindcss#2273] [Help] [answered]
I have used tailwind config for adding a background image into my next js project but when I use the bg-image utility, the image is not shown on the website. Here is the code of my project, kindly provide any solution of how can I get the background image. @adamwathan 

Code in  Tailwind config
`{
      backgroundImage: (theme) => ({
        "main-banner": "url('/public/images/homescreenBg.jpg')"
      }),`

Code used in my project
`<div className="w-full h-full bg-main-banner">`

**Comment by amdrew** [comment, marked as answer]
@kartishp Since it's a next.js project, try referencing the image without `/public`. See: https://nextjs.org/docs/basic-features/static-file-serving

Otherwise, it might be helpful to post your full `tailwind.config.js` file. Check it matches the following example, so it's placed inside `extend`: 
https://tailwindcss.com/docs/background-image

```js
 // tailwind.config.js
  module.exports = {
    theme: {
      extend: {
        backgroundImage: theme => ({
         'hero-pattern': "url('/img/hero-pattern.svg')",
         'footer-texture': "url('/img/footer-texture.png')",
        })
      }
    }
  }
```

Good luck!

**Reply by kartishp** [reply]
@amdrew  can you help me out with the positioning and sizing of the background image in `tailwind.config.js` file in `backgroundImage` utility itself

**Reply by larts85** [reply]
This or none other solution works for me, i've done everithing that official docs saits without good results, i can see the property `background-image: url('/img/bg-footer.png');` and also when I hover it show me the thumbnail, but still getting blanck on the page 🤔🥺🤯

**Reply by blazk0** [reply]
putting the assets folder inside public folder worked. 

Thanks a ton. Maybe add a hint in the documentation about this ?

**Comment by globalexport** [comment]
@larts85  Did your restart/rebuild your application after making changes? 
This was necessary in my Vite project.
